ApiTests/VelocityApiTests.m

```diff
@@ -55,9 +55,9 @@ - (void)testCapture
     XCTestExpectation *expectation = [self expectationWithDescription:@"APIResponse"];
     __block NSError *apiError = nil;
 
-    [self.apiClient uploadForTracking:capture success:^{
+    [self.apiClient uploadForTracking:capture success:^(NSUInteger bytesSent){
```

Comment:
Imho, This ApiTests target should be removed as it is kind of integration test and now with web sockets update it is totally outdated.